deployment/terraform/assets/.terraform.lock.hcl

```diff
@@ -2,21 +2,81 @@
 # Manual edits may be lost in future updates.
```

Comment:
Is this file meant to be checked in?

Reply:
![image](https://github.com/mattermost/mattermost-load-test-ng/assets/3924815/62790849-42dc-40e9-ba9f-bad6db68d057)

You asked this exact same question almost [three years ago](https://github.com/mattermost/mattermost-load-test-ng/pull/457#discussion_r634477970) hahaha

Answering your question: yes. I had the same question you had three years ago (and today), double-checked the docs and yes, they still recommend to check it in.

Reply:
:facepalm: